apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -322,7 +322,7 @@ export function initializeSeenParam(seen: string): boolean | null {
   let isSeen: boolean = null;
 
   if (seen) {
-    isSeen = seen == 'true';
+    isSeen = seen === 'true';
```

Comment:
@djabarovgeorge is this not strict check is intentional here?

Reply:
I think it's intentional as its comparing the string value to the boolean value 

![image](https://user-images.githubusercontent.com/72117025/208017491-d9c61cf8-9300-401d-95f1-41da725df776.png)

should I revert them back?

Reply:
this is interesting since seen is being compared to a string while also being a string so using a strict equality check would be the better option IMO so "===". Using == does type coercion which converts both the values being compared to the same type e.g 1 == '1' would be true but 1==='1' would be false.

Reply:
You guys are right it was intentional, but we can remove this function altogether because nobody uses it now, due to this reason that we came up with an object query solution in the end. Sorry, my mistake not removing it.

Reply:
> You guys are right it was intentional, but we can remove this function altogether because nobody uses it now, due to this reason that we came up with an object query solution in the end. Sorry, my mistake not removing it.

@djabarovgeorge should I remove that function then?

Reply:
Let's keep the old functionality now and restore the '==' non strict check. Will properly deprecate it over the next few releases.

Reply:
@scopsy should all the strict checks should be restored OR 
 
![image](https://user-images.githubusercontent.com/72117025/208138174-063e7705-84ca-4484-8539-b17a415473d5.png)
this one